File: codeup/100000574/F.cpp
#include <iostream>
using namespace std;

int main() {
  int N, num, sum;
  while (scanf("%d", &N) != EOF) {
    sum = 0;
    while (N--) {
      scanf("%d", &num);
      sum += num;
    }
    printf("%d\n", sum);
  }

  return 0;
}

File: codeup/100000574/F.py
#!/usr/bin/python3
import sys

while True:
    line = sys.stdin.readline()
    if not line:
        break
    num_list = [int(x) for x in line.split()]
    sum = 0
    for num in num_list[1:]:
        sum += num
    print(sum)
